README.md

```diff
@@ -1 +1,71 @@
-# To_do_list
\ No newline at end of file
+# To_do_list
+
+> In this project we created a simple application wich will allow users to add their daily tasks 
+and can even restore them. They can remove a specific task by clicking on the right icon of the task or remove 
+all completed tasks by clicking on the clear-all-completed button at the bottom of the apllication.
+
+## How it looks like :
+
+![Todolist2](https://user-images.githubusercontent.com/94958024/161342026-0f2728c0-46b4-4fc1-a51c-b9f7612e6a81.png)
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- webpack
+
+## Live Demo is avaible here
+
+[Live Demo Link](https://anicetfantomas.github.io/To_do_list/)
+
+
```

Comment:
The live demo link does not contain all the expected functionalities, the user should be able to add a new task and clear all completed. Kindly fix it